packages/ndla-ui/src/Image/Image.tsx

```diff
@@ -38,22 +38,30 @@ const getSrcSet = (src: string, crop: ImageCrop | undefined, focalPoint: ImageFo
   return widths.map((width) => `${src}?${makeSrcQueryString(width, crop, focalPoint)} ${width}w`).join(', ');
 };
 
-interface StyledImageWrapperProps {
-  svg?: boolean;
-}
-
-const StyledImageWrapper = styled.div<StyledImageWrapperProps>`
+const StyledImageWrapper = styled.div`
   position: relative;
-  ${({ svg }) => {
-    return (
-      svg &&
-      css`
-      display: flex
-      justify-content: center;
-    `
-    );
-  }}
+
+  &[data-svg='true'] {
+    display: flex;
+    justify-content: center;
+  }
+  &[data-ingrid='true'] {
+    border: 1px solid ${colors.brand.tertiary};
+    border-bottom: 0;
+    border-radius: 4px;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+  }
```

Comment:
I og med at nye funksjoner i ed serialiserer `inGrid` til `in-grid` burde nok dette være `data-in-grid`